src/services/transaction-service/transaction-service.service.ts

```diff
@@ -14,12 +15,22 @@ export class TransactionService {
     trusted?: boolean,
     excludeSpam?: boolean,
   ): Promise<Balance[]> {
-    const url = this.baseUrl + `/api/v1/safes/${safeAddress}/balances/usd/`;
+    const url = `${this.baseUrl}/api/v1/safes/${safeAddress}/balances/usd/`;
     try {
-      const { data } = await this.httpService.axiosRef.get(url, {
+      const res = await this.httpService.axiosRef.get(url, {
         params: { trusted: trusted, excludeSpam: excludeSpam },
       });
-      return data;
+      return res?.data;
```

Comment:
Is there a way to deal with `res` having `any` type? It looks like this function could return `undefined` but it is not reflected in the signature.

Reply:
The function shouldn't return `undefined`. In fact if `res.data` is not present it should throw an error, I'll back to the old implementation, thanks!